src/executor.ts

```diff
@@ -34,6 +34,7 @@ interface RunOptions<T> {
   timeout: number;
   valueSerializer: Serializer<T>;
   errorSerializer: Serializer<Error>;
+  ignoreErrors?: boolean;
```

Comment:
I’d suggest making this more flexible by using a callback function instead of a boolean flag. This would allow users to selectively ignore certain errors.
```suggestion
  shouldIgnoreError?: (error: Error) => boolean;
```
By default, it could be `() => false`, but users would have the option to override it with custom logic.